syntax = "proto3";

package user;

service UserService {
  rpc GetUser (UserRequest) returns (UserReply) {}
  rpc GetUserByID (UserIDRequest) returns (UserReply) {}
}

message UserRequest {
  string login = 1;
}

message UserIDRequest {
  string user_id = 1;
}

message UserReply {
  string id = 1;
  string login = 2;
  string email = 3;
}